guardian/typerighter
# Move gu-cdk for rule management app to Patterns

## What does this change?

Moves the gu-cdk for the rule management app to the new `Patterns` classes. This significantly reduces the boilerplate required.

## How to test

This should be a no-op – the service should continue to run as normal.

To test, you could try running `yarn synth` in the `./cdk` folder, and cloudforming the resulting stack in `./cdk.out`. The result should be a working service.

Tested in CODE.

## How can we measure success?

Reduced LoC and more comprehensible cdk.
